components/org.wso2.carbon.identity.authz.valve/src/main/java/org/wso2/carbon/identity/authz/valve/AuthorizationValve.java

```diff
@@ -67,6 +67,8 @@ public void invoke(Request request, Response response) throws IOException, Servl
                 authorizationContext.setIsCrossTenantAllowed(resourceConfig.isCrossTenantAllowed());
                 authorizationContext.setAllowedTenants(resourceConfig.getCrossAccessAllowedTenants());
             }
+            // TODO: Authorization for requests containing '/o/' path param should be handled via the new role management
```

Comment:
Let's create an issue and remove the todo.

Reply:
https://github.com/wso2/product-is/issues/13955